.clang-format

```diff
@@ -12,7 +12,7 @@ IndentPPDirectives: BeforeHash
 IndentWidth: 4
 LambdaBodyIndentation: OuterScope
 PointerAlignment: Left
-SortIncludes: false
```

Comment:
I'd probably recommend against changing this line in `clang-format`, as I've run into cases where it will have unintended side effects that affect compilation.

Here's one example from `MaterialXFormat/File.cpp`, where the current include order correctly places `windows.h` and `unistd.h` first in their respective blocks:

```
#if defined(_WIN32)
    #define WIN32_LEAN_AND_MEAN
    #include <windows.h>
    #include <direct.h>
#else
    #include <unistd.h>
    #include <sys/stat.h>
    #include <dirent.h>
#endif
```

But with `SortIncludes` set to `true`, this gets reformatted to the following:

```
#if defined(_WIN32)
    #define WIN32_LEAN_AND_MEAN
    #include <direct.h>
    #include <windows.h>
#else
    #include <dirent.h>
    #include <sys/stat.h>
    #include <unistd.h>
#endif
```